default.yml

```diff
@@ -10,7 +10,7 @@ Layout/AccessModifierIndentation:
   EnforcedStyle: outdent
 
 Layout/EmptyLinesAroundAttributeAccessor:
-  Enabled: false
+  Enabled: true
```

Comment:
🗣️ This seems draconian. Is the a Bad Thing?
```
    ATTRIBUTES = %i(attr1 attr2 attr3).freeze
    attr_reader(*ATTRIBUTES)
```

Reply:
Honestly if this came up on a PR I would ask for a blank line between constants in general (even this one) and macros

Reply:
This is just a default, you can always override it if you get buy-in from your team for a particular use case.

Reply:
But if there's only one of each in a class, and they go together... seems like a silly thing to generalize.

But I'll defer to whatever consensus forms.

Reply:
I think that is a relatively uncommon case, compared to the vast majority of ruby objects using attr_reader/attr_accessor

And I say that as someone who has written at least as half dozen classes with exactly this pattern.